app/views/layouts/application.html.erb

```diff
@@ -11,13 +11,13 @@
   </head>
 
   <body>
-    <header>
-      <%= render "shared/navbar" %>
-    </header>
-
-    <main>
-      <%= render "devise/shared/flashes" %>
-      <%= yield %>
-    </main>
+  <header>
+    <%= render "shared/navbar" %>
+  </header>
+    <p class="notice"><%= notice %></p>
+    <p class="alert"><%= alert %></p>
+    <%= render "devise/shared/flashes" %>
+    <%= yield %>
+    <%= render "shared/footer" %>
   </body>
```

Comment:
This still isn't correct
You need to keep master as it was and add your footer.

Reply:
done